src/SolrClient.php

```diff
@@ -14,13 +14,14 @@
 namespace iCom\SolrClient;
 
 use iCom\SolrClient\Client\SymfonyClient;
+use Symfony\Component\HttpClient\HttpClient;
 
-class SolrClient
+final class SolrClient
 {
     public static function create(array $options): Client
     {
         if (class_exists(SymfonyClient::class)) {
-            return new SymfonyClient($options);
+            return new SymfonyClient(HttpClient::create($options));
         }
 
         throw new \LogicException('You need to install a client implementation.');
```

Comment:
You need to install an HTTP client implementation. Try running "composer require symfony/http-client".